corehq/apps/api/resources/v0_5.py

```diff
@@ -489,3 +495,30 @@ def dehydrate(self, bundle):
         bundle.data['product_name'] = bundle.obj.sql_product.name
         bundle.data['transaction_date'] = bundle.obj.report.date
         return bundle
+
+
+class SimpleReportConfigurationResource(CouchResourceMixin, HqBaseResource, DomainSpecificResourceMixin):
```

Comment:
i see the spec also mentions exposing an endpoint that runs the ucr report - is that here too? it seemed like this was just listing the configuration of the report unless i'm missing something